server/src/utils/directoryContains.ts

```diff
@@ -1,10 +1,13 @@
 import path from "path";
 
+/**
+ * Returns true if testPath is equal to dirPath or a subdirectory of it
+ * @param dirPath
+ * @param testPath
+ * @returns
+ */
 export function directoryContains(dirPath: string, testPath: string): boolean {
   const relative = path.relative(dirPath, testPath);
-  return (
-    !!relative &&
-    !relative.startsWith(`..${path.sep}`) &&
-    !path.isAbsolute(relative)
-  );
+
+  return !relative.startsWith(`..${path.sep}`) && !path.isAbsolute(relative);
```

Comment:
Is there any danger here from os separator differences? Path relative is going to return os specific separators.

Reply:
You are right. That's why `path.sep` is used, so it works across OSs.

Reply:
Ah, I missed that. Cool